docs/book/src/component-config-tutorial/testdata/project/config/rbac/kustomization.yaml

```diff
@@ -16,3 +16,6 @@ resources:
 - auth_proxy_role.yaml
 - auth_proxy_role_binding.yaml
 - auth_proxy_client_clusterrole.yaml
+# Editor and Viewer roles for each CRD to be used by end users.
```

Comment:
Can we add a better comment here to clarify why it is scaffold/when it is useful?
Example/Suggestion

```
For each CRD, "Editor" and "Viewer" roles are scaffolded by default, aiding admins in cluster management. While optional for managers, who can modify or remove them, their removal means they won't be installed with your solution.
```